### PR TITLE
feat: added all products component

### PR DESCRIPTION
**Pull request by mansi2011**
added home and all product page

chrome-capture

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&family=Roboto:wght@400;700&display=swap');
-
+@import url("/Component/Products/product.css");
 
 *{
```
Comment (MaruthiPothuganti) [inline styles.css:4]: Generally colors are defined in root selector

File: Component/Products/product.html [hunk 2]
```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initiak-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>e-shop - products | e-commerce website</title>
```
Comment (MaruthiPothuganti) [inline Component/Products/product.html:8]: This is a different title than that is present  on the homepage

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Everything else looks good

File: Component/Products/product.html (hunk 2 above)
Comment (mansi2011) [reply Component/Products/product.html:8]: yes. It should be different na for different pages? Isn't it?

File: styles.css (hunk 1 above)
Comment (mansi2011) [reply styles.css:4]: i did defined in root only. can you check again

**Review by spraveenofficial — APPROVED** [review]

File: Component/Products/product.html (hunk 2 above)
Comment (MaruthiPothuganti) [reply Component/Products/product.html:8]: Yeah

File: styles.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply styles.css:4]: Oh okay. Sorry, I got confused

**Pull request merged**